packages/elements/src/number-field/__test__/number-field.step.test.js

```diff
@@ -220,4 +220,43 @@ describe('number-field/Step', function () {
       await expectValues(el, [-1, -2, -3, -4, -5], UP, -1.5);
     });
   });
+
+  describe('Support long press', function () {
+    let el;
+    const dispatchLongTapEvent = async (el, wait) => {
+      setTimeout(() =>
+        el.dispatchEvent(
+          new Event('tapstart', {
+            bubbles: true
+          })
+        )
+      );
+      await oneEvent(el, 'tapstart');
+      if (wait) {
+        await aTimeout(wait);
+      }
+      setTimeout(() =>
+        el.dispatchEvent(
+          new Event('tapend', {
+            bubbles: true
+          })
+        )
+      );
+      await oneEvent(el, 'tapend');
+    };
+
+    beforeEach(async function () {
+      el = await fixture('<ef-number-field></ef-number-field>');
+    });
+    it('By tap, Step Up', async function () {
+      await dispatchLongTapEvent(el.spinnerUpEl, 1000);
+      await elementUpdated(el);
+      expect(Number(el.value)).to.greaterThan(0);
```

Comment:
I think`greaterThan(0)` and `lessThan(0)` here aren't cover the long press spinner use case.
It could pass this test case by 1 or -1 values from just a single tap

Reply:
Agree with you. So I update it to expect greaterthan and lessthan to 3 and -3 accordingly.